src/lib/Uploader.svelte

```diff
@@ -56,10 +56,10 @@
     .curator {
       font-size: 0.75rem;
       color: $color-muted-text;
-      visibility: hidden;
+      display: none;
 
       @media (min-width: 992px) {
```

Comment:
This is not part of the PR, but I've noticed that the logic for displaying the curators seems off.
They get displayed as long we are in two-column layout. But as soon as it switches to one-column layout (and has a real lot of space) it vanishes. That doesn't seem sensible.
Maybe that calls for container queries? Same in #109. It works great there, but they **may** be a more direct way to express the desired outcome and as a result more robust?

Reply:
Yeah it could probably stick around a bit longer before disappearing on smaller screens. Didn't pay much attention to that in this PR, just wanted it to stay consistent with how it was before.